Lpd 5780

wanderlast opened the pull request.

mauricesepe committed:
LPD-6987 Add object-api for SugarCRM connector

mauricesepe committed:
LPD-6987 SF

mauricesepe committed:
LPD-6987 language keys + buildLang

mauricesepe committed:
LPD-6987 Implement WebCacheItem for Sugarcrm connector

mauricesepe committed:
LPD-6987 SF

mauricesepe committed:
LPD-6987 Implement SugarcrmObjectEntryManagerImpl

mauricesepe committed:
LPD-6987 Implementation + SF

brianchandotcom committed:
LPD-6987 Rename

brianchandotcom committed:
LPD-6987 Rename

brianchandotcom committed:
LPD-6987 Sort

brianchandotcom committed:
LPD-6987 We need to cache the full key

brianchandotcom committed:
LPD-6987 Rename

brianchandotcom committed:
LPD-6987 SF

brianchandotcom committed:
LPD-6987 Do we really need && ?

brianchandotcom committed:
LPD-6987 Sort

brianchandotcom committed:
LPD-6987 Inline

brianchandotcom committed:
LPD-6987 SF

brianchandotcom committed:
LPD-6987 SF

mauricesepe committed:
LPD-6987 Remove unnecessary &

mauricesepe committed:
LPD-6987 Initial commit for test class

wanderlast committed:
LPD-5780 - functional test for COMMERCE-13162

mauricesepe committed:
LPD-6987 Remove unnecessary logs

mauricesepe committed:
LPD-6987 Fix bug with base URL not appending the options correctly

mauricesepe committed:
LPD-6987 Implement updateObjectEntry + SF

mauricesepe committed:
LPD-6987 Implement test class for SugarCRMObjectEntryManagerImplTest

wanderlast committed:
LPD-5780 - refactor, separate product relations related functionality into its own page

wanderlast committed:
LPD-5780 - SF

wanderlast committed:
LPD-5780 - update imports to adjust for changes made elsewhere

wanderlast committed:
LPD-5780 - further updates to bring in line with other tests

wanderlast committed:
LPD-5780 - SF

Commit:
LPD-19574 - Allow creation of object definitions that know it's root definition

# breaking

## What modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

 a. added rootObjectDefinitionId to addObjectDefinition method

## Why

 a. It became necessary to create objectDefinitions that know who is
their rootObjectDefinition during creation to allow imports

----

Commit:
LPD-19574 - Validate root node before posting object definitions

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

wanderlast committed:
LPD-5780 - combine redundant files

brittneyq committed:
LRCI-4116 Remove redundant propeties and add condition for quarterly release branches

guilhermedcamacho committed:
LPD-6987 Fix pagination logic

guilhermedcamacho committed:
LPD-6987 Move toJSONObject method to base class and reuse

Commit:
LPD-19574 - Create method to allow importing edge relationships

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19574 Make sure tests do not create root objects without intention

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

guilhermedcamacho committed:
LPD-6987 Provides a way to perform specific logic on each object field

guilhermedcamacho committed:
LPD-6987 Move toObjectEntry method to base class and reuse

guilhermedcamacho committed:
LPD-6987 Provides a generic way to pass default object field names

mauricesepe committed:
LPD-6987 Initial implementation of FilterFactory

mauricesepe committed:
LPD-6987 Implement changes for SugarQuery

mauricesepe committed:
LPD-6987 SF

Commit:
LPD-19575- Validate object definition status before updating

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19575- Create a method to disable edge relationships for objects in diferent root models

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19575- Enable putObjectDefinition to handle edge relationships

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

antonio-ortega committed:
LPD-19992 Update liferay-ckeditor version to v4.21.0-liferay.2

antonio-ortega committed:
LPD-19992 Regen yarn.lock file

Commit:
LPD-19575- Start unbind using node as root from subtree

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

guilhermedcamacho committed:
LPD-6987 Fix field type

markocikos committed:
LPD-20604 Include Playwright in relevant tests

antonio-ortega committed:
LPD-20691 This was added because IE11 was unable to handle the situation when we had two different favicon and navigate using SPA. Since we no longer support IE11, we can get rid of it, letting this resource to be cached and improve performance

brianikim committed:
LPD-19944 User can be null. Use fetch instead.

brianikim committed:
LPD-19944 We need to use current user for audits.

Commit:
LPD-19574 - Add reindex to update methods for objectDefinitions

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

markocikos committed:
LPD-20604 Standardize project name

Commit:
LPD-19575- Show error message when failing import because of status

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

brianikim committed:
LPD-21146 Guest orders need to be reindexed on scheduled deletion

izaera committed:
LPD-18498 Fix first rendering in widget pages editor

marianoalvarosaiz committed:
LPD-21199 Do not set CompanyThreadLocal at BufferedIncrementRunnable level. BufferedIncrementRunnable executes tasks from the BatchablePipe queue, but this queue can have method invocations related to different companies.
This works without DB Partition (view count increment for example) because the method to invoke includes the companyId, so, while the CompanyThreadLocal may be incorrect, since there is only one table (ViewCountEntry) common to every company the value will be inserted correctly.
For DB Partition the CompanyThreadLocal must be correct because it is used to select the partition, so it is not enough with the method invocation companyId parameter.

marianoalvarosaiz committed:
LPD-21199 Set the companyId at BufferedIncreasableEntry level. This is the real method invocation and it can save the real current companyId that would have been used if no BufferedIncrement advice existed.

alessiorendina committed:
LPD-20133 Added release type feature flag

alessiorendina committed:
LPD-20133 Added title and description

alessiorendina committed:
LPD-20133 Update usages

ccorreagg committed:
LPD-21241 Add GraphQL tests to verify the new GraphQL namespace

ccorreagg committed:
LPD-21241 buildREST

marianoalvarosaiz committed:
LPD-21199 Create test method to ensure that when executing a bufferable method the CompanyThreadLocal has a correct value.

matyaswollner committed:
LPD-12805 - Add back url title to User Admin

ccorreagg committed:
LPD-21241 Generate tests for namespaces only for compatibilityVersion >= 5

ccorreagg committed:
LPD-21241 Update compatibilityVersion parameter

MarinhoFeliphe committed:
LPD-18757 Disable script management configuration on LXC environment

MarinhoFeliphe committed:
LPD-18757 Semver

ccorreagg committed:
LPD-21241 Fix test

ccorreagg committed:
LPD-21241 Fix test

ccorreagg committed:
LPD-21241 Fix test

Commit:
LPD-19574- buildService

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19574 - Update addObjectDefinition calls to match new parameter

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19574- Expose updateObjectDefinitionPortlet method

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19574- buildService

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

abfranca committed:
ISSD-3199 MDF Activity Expired inside MDF Claim form

Commit:
LPD-19574 - buildService

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19574 - buildService

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19574 - Allow edge relationships to be created during ObjectDefinition import

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19575- buildService

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

MarinhoFeliphe committed:
LPD-18757 Disable script content creation and execution on LXC SaaS mode

danvb112 committed:
LPD-21575 Use Clay table instead of a list in the ModalImportWarning.tsx

danvb112 committed:
LPD-21575 Send a FormData instead of a JSON array to backend in modal to import and Object Definition

abfranca committed:
ISSD-3199 Adjusting exibition logic

wanderlast committed:
LPD-5780 - resync

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

wanderlast commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-36.liferay.com/job/test-portal-source-format/2980/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/2980/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b6b6d459a89fff752a17d825beafa4e0e884cc52">b6b6d459a89fff752a17d825beafa4e0e884cc52</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/wanderlast/liferay-portal/tree/LPD-5780">LPD-5780</a><br/>Branch GIT ID: <a href="https://github.com/wanderlast/liferay-portal/commit/f921e5f3a987af806e5e0415f693658c9b13e902">f921e5f3a987af806e5e0415f693658c9b13e902</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-36.liferay.com/job/test-portal-source-format/2980/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/2980/jenkins-report.html">here</a>.</h5><pre>     [exec] 
     [exec] added 2646 packages, and audited 2647 packages in 14s
     [exec] 
     [exec] 210 packages are looking for funding
     [exec]   run `npm fund` for details
     [exec] 
     [exec] 71 vulnerabilities (4 low, 39 moderate, 24 high, 4 critical)
     [exec] 
     [exec] To address issues that do not require attention, run:
     [exec]   npm audit fix
     [exec] 
     [exec] To address all issues (including breaking changes), run:
     [exec]   npm audit fix --force
     [exec] 
     [exec] Run `npm audit` for details.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat
     [exec] 
     [exec] &gt; @liferay/playwright@1.0.0 checkFormat
     [exec] &gt; tsc &amp;&amp; liferay-npm-scripts lint &amp;&amp; liferay-npm-scripts format:check
     [exec] 
     [exec] 
     [exec] pages/commerce/commerceAdminProductDetailsProductRelationsPage.ts: BAD
     [exec] Prettier checked 272 files, 1 file has problems
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] Gradle build finished at 2024-03-27 02:45:17.426.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/modules/test/playwright/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] &gt; Run with --info or --debug option to get more log output.
     [exec] &gt; Run with --scan to get full insights.
     [exec] &gt; Get more help at https://help.gradle.org.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$1(ExecuteActionsTaskExecuter.java:148)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:282)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:146)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:134)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-36.liferay.com/job/test-portal-source-format/2980/">test-portal-source-format#2980</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/2980/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/wanderlast/liferay-portal/pull/109">wanderlast#109</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2555310418">[master] ci:test:sf - wanderlast &gt; wanderlast - PR#109 - 2024-03-26[19:40:09]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>2555310418<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-36.liferay.com/job/publish-testray-report/12280/">publish-testray-report#12280</a><br/></div></div>

abfranca committed:
ISSD-3199 Edit activity status text on mdf request page

matyaswollner committed:
LPD-21201 - Add account contacts sub-endpoints(yaml)

matyaswollner committed:
LPD-21201 - Add account contacts sub-endpoints

matyaswollner committed:
LPD-21201 - Semantic versioning

matyaswollner committed:
LPD-21201 - Autogenerated code (buildRest)

abfranca committed:
ISSD-3199 Changing exibition logic for new claim button

Commit:
LPD-19574- Avoid null edge values when importing

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-7012 - Add functional tests

larissacribeiro committed:
LPD-21684 Add FieldBase helpMessage property to InputLocalized

abfranca committed:
ISSD-3199 Renaming constants

michaelhashimoto committed:
LRCI-4149 Add ability to delete a job using Liferay REST API

michaelhashimoto committed:
LRCI-4149 Add job definition & job definition parameter liferay objects

michaelhashimoto committed:
LRCI-4149 Add Job Definition parameter entities

michaelhashimoto committed:
LRCI-4149 Add Job Definition entities

michaelhashimoto committed:
LRCI-4149 SF

michaelhashimoto committed:
LRCI-4149 Add js files that will enable interactions with job definition & parameters

michaelhashimoto committed:
LRCI-4149 Get the job definitions from Liferay REST API instead of Spring

michaelhashimoto committed:
LRCI-4149 Add a method to create a job page

michaelhashimoto committed:
LRCI-4149 Shift away from spring boot to liferay rest api

michaelhashimoto committed:
LRCI-4149 Remove unused spring boot data

michaelhashimoto committed:
LRCI-4149 Fix after testing

brianikim committed:
LPD-19944 Fix Tests

danvb112 committed:
LPD-21575 Export errorsUtils from object-js-components-web

danvb112 committed:
LPD-21575 Create ModalImportFailed component

danvb112 committed:
LPD-21575 Adapt ModalImport in case of failure when importing multiple definitions

danvb112 committed:
LPD-21575 Add Language keys

danvb112 committed:
LPD-21575 Add loading to import button

rebsilva committed:
LPD-2275 Fix construction of the timers sidebar when adding a ScriptedReassignment timer

danvb112 committed:
LPD-21575 Use the JSON Stingfy method in case of receiving an array from the response

istvansajtos committed:
LPD-6874 Add the right portlet to the utility page

istvansajtos committed:
LPD-6875 Add the right portlet to the utility page

guilhermedcamacho committed:
LPD-6987 Move service to base class

guilhermedcamacho committed:
LPD-6987 They are already a JSONObject

guilhermedcamacho committed:
LPD-6987 Improve test to cover and/or operators for filters

guilhermedcamacho committed:
LPD-6987 SF

alessiorendina committed:
LPD-20133 BuildLang

lfbesada committed:
LPD-21831 Adds support to create a ServiceContextAutoCloseable specifying the user

lfbesada committed:
LPD-21831 Baseline

lfbesada committed:
LPD-21831 SF, company never is null when invoking this method

lfbesada committed:
LPD-21831 Uses LayoutServiceContextHelper to ensure the adequate ServiceContext when processing the file

lfbesada committed:
LPD-21831 SF, remove redundancies since the ServiceContextAutoCloseable will take care of all this

Commit:
LPD-7012 - Check if trigger is visible before attempting to click it in clickAndExpectToBeVisible.

Commit:
LPD-7012 - npm run format.

Commit:
LPD-7012 - Wait for page to reload after editing the client extension to guarantee form submitted.

guilhermedcamacho committed:
LPD-6987 Move getDateFormat and toObjectEntries methods to base class and reusea

michaelhashimoto committed:
LRCI-4149 SF

Commit:
LPD-7012 - Rename from globalJS to editJS.

Commit:
LPD-7012 - Rename newEditorConfigContributor to editEditorConfigContributor.

Commit:
LPD-7012 - Move page attributes shared by all client extensions to a shared EditClientExtensionsPage.

Commit:
LPD-7012 - Have EditEditorConfigContributorPage extend from EditClientExtensionsPage.

Commit:
LPD-7012 - Rename ThemeCSSClientExtensionsPage to EditThemeCSSClientExtensionsPage.

Commit:
LPD-7012 - Have EditThemeCSSClientExtensionsPage extend from EditClientExtensionsPage.

Commit:
LPD-7012 - Rename globalJS to js.

Commit:
LPD-7012 - Remove unused locators.

Commit:
LPD-7012 - Prefer randomString over a constant unique names to reduce test flakiness.

Commit:
LPD-7012 - Prefer getByRole.

Commit:
LPD-7012 - SF and wordsmithing.

Mikellorza committed:
LPD-21854 Do not translate localized title for successfully message

Commit:
LPD-19574- Update updateObjectDefinitionPortlet method signature

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19574- buildService

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

Commit:
LPD-19574- Update method calls

Signed-off-by: Guilherme Sa <guilherme.sa@liferay.com>

thaynam committed:
ISSD-3940 Add log to refused accounts in Partner Portal's auto-provisioning

achaparro committed:
LPD-21199 Manual SF

antonio-ortega committed:
LPD-20691 Fix test

Mikellorza committed:
LPD-21925 Remove FF

Mikellorza committed:
LPD-21925 SF

nNilton committed:
LPD-21840 assert layout type was changed after site update

michaelhashimoto committed:
LRCI-4149 Consolidate ordered batch cx into 1 cx

michaelhashimoto committed:
LRCI-4149 Update reference to the asah release branch

Commit:
LPD-21932 Add Tier Prices to App Review Page

amosfong committed:
ISSD-3940 SF

guilhermedcamacho committed:
LPD-6987 Create ObjectEntryManagerHttp interface and BaseObjectEntryManagerHttp class

guilhermedcamacho committed:
LPD-6987 Implement ObjectEntryManagerHttp and extend BaseObjectEntryManagerHttp to reuse code

lfbesada committed:
LPD-21831 Auto

brianchandotcom committed:
LPD-20495 Wordsmith

brianchandotcom committed:
LPD-20495 Better one

brianchandotcom committed:
LPD-20495 Inline

brianchandotcom committed:
LPD-20495 SF

brianchandotcom committed:
LPD-20495 Rename

brianchandotcom committed:
LPD-20495 Rename

brianchandotcom committed:
LPD-20495 auto SF

smottal committed:
LPD-20130 Update FF to release

guilhermedcamacho committed:
LPD-6987 Baseline

guilhermedcamacho committed:
LPD-6987 Ignore test while credentials are being configured in CI environment

ccorreagg committed:
LPD-15236 breaking changes amend

Commit:
LPD-21935 Rename

Commit:
LPD-21869 Increase the individuals interactions with the page

Commit:
LPD-21869 Modify the test to check if the empty state appears or not

MumenTayyem committed:
LPD-19870 - change - to _

kevenleone committed:
LPD-21935 Fallback redirect

kevenleone committed:
LPD-21935 Split Component

kevenleone committed:
LPD-21935 Rename

Commit:
LPD-22027 Add a wait for requests to load before disabled the toggle to avoid errors

ealonso committed:
ISSD-3954 Removes functional and duplicated stable suites from relevant

veroglez committed:
LPD-20159 Remove unnecessary test since the show/hide fragment button does not appear for master page items

welitonpe committed:
LPD-22033 Fixed Creation of SKUs on Create App Flow, and some Bugs on GetApp Flow

kenjiheigel committed:
LRCI-4156 Use nearest parent gradlew

kevenleone committed:
LPD-22033 Trial option is hidden if SKU is not purchasable

kevenleone committed:
LPD-22033 Display only the purchasable skus

kevenleone committed:
LPD-22033 Standardize

igor-franca committed:
LPD-1952 Remove disableFlag property from InputLocalized

# breaking

## What modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/input/InputLocalized.tsx

modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/input/InputLocalized.tsx now it does not receive the disableFalg property;

## Why

this change is required to improve the usability of the InputLocalized component

----

danvb112 committed:
LPD-21575 buildlang

carolmariaabb committed:
LPD-22098 Include user email address and user id in the audit request threadLocal for batch engine execution

thiagobuarque-liferay committed:
LPD-7012 wait for the globaljs client extension checkbox to have the click event

julschong committed:
LPD-20674 Fix Test, use (1-12) hour "h" instead of (0-11) hour "K"

kevenleone committed:
LPD-21932 No longer dispatches app

gabrielwas committed:
LPD-22098 SF

brianchandotcom committed:
LPD-22098 Sort

ryanschuhler committed:
ISSD-3192 Fix page-definitions

kenjiheigel committed:
LRCI-4063 Add new report frontend files

MumenTayyem committed:
LPD-19870 - change - to _ #2

brianchandotcom committed:
LPD-12805 Match the other files that were modified in this PR

brianchandotcom committed:
LPD-21575 I'm not a big fan of (s). Make 2 keys if necessary.

brianchandotcom committed:
LPD-21575 Regen

brianchandotcom committed:
LPD-19575 Validate before adding anything

brianchandotcom committed:
LPD-19575 Wordsmith

brianchandotcom committed:
LPD-11274 Call it fetch

brianchandotcom committed:
LPD-11274 auto SF

brianchandotcom committed:
LPD-11274 Call it fetch

brianchandotcom committed:
LPD-11274 auto SF

brianchandotcom committed:
LPD-11274 Rename

brianchandotcom committed:
LPD-11274 Wrong package

brianchandotcom committed:
LPD-11274 An OSGi private component should never go to the outside world to get another one of its own private components

brianchandotcom committed:
LPD-11274 Make Salesforce match Sugar

brianchandotcom committed:
LPD-11274 I wish the Salesforce and Sugar tests were more similar

brianchandotcom committed:
LPD-11274 Rename

brianchandotcom committed:
LPD-11274 Rename

brianchandotcom committed:
LPD-11274 Simplify

brianchandotcom committed:
LPD-11274 This should be the same thing (without casting)

brianchandotcom committed:
LPD-11274 Call get just once

brianchandotcom committed:
LPD-11274 Harmonize Sugar and Salesforce

brianchandotcom committed:
LPD-11274 Harmonize Sugar and Salesforce

brianchandotcom committed:
LPD-11274 Use our trim (faster)

brianchandotcom committed:
LPD-11274 Inline

brianchandotcom committed:
LPD-11274 SF

brianchandotcom committed:
LPD-11274 Simplify

brianchandotcom committed:
LPD-11274 Simplify

brianchandotcom committed:
LPD-11274 Do we need this?

brianchandotcom committed:
LPD-11274 Document TODO

brianchandotcom committed:
LPD-11274 Rename

brianchandotcom committed:
LPD-11274 SF

brianchandotcom committed:
LPD-11274 SF

brianikim committed:
LPD-21146 Fallback to default if fetchUser returns null

fetchUser can potentially return null with scheduler events.

brianchandotcom committed:
LPD-11274 Clarity over brevity

brianikim committed:
LPD-21146 Autogenerated Code (buildService)

brianchandotcom committed:
LPD-11274 Use optimized caching

brianchandotcom committed:
LPD-11274 Follow the order in object-service/src/main/resources/META-INF/sql/tables.sql

brianchandotcom committed:
LPD-19944 CentralizedThreadLocal.clearShortLivedThreadLocals will clear PermissionThreadLocal.setPermissionChecker and CompanyThreadLocal.setCompanyId

lfbesada committed:
LPD-22117 Update the SEO configuration category icon

brianchandotcom committed:
LPD-11274 Regen

brianchandotcom committed:
LPD-21241 SF

brianchandotcom committed:
LPD-21241 prep next

brianchandotcom committed:
LPD-21669 prep next

brianchandotcom committed:
LPD-21241 LPD-21669 prep next

brianchandotcom committed:
LPD-21201 Missing @Override

Commit:
LPD-7012 - No need to have two checks that the iframe is ready.

Commit:
LPD-7012 - Rename variable and add comment to explain the purpose of checking for 'Select Items'.

izaera committed:
LPD-18498 Sort

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPD-18498 SF

wanderlast commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a3f4976407775eab5d00ef86db452121dab1414d">a3f4976407775eab5d00ef86db452121dab1414d</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/wanderlast/liferay-portal/tree/LPD-5780">LPD-5780</a><br/>Branch GIT ID: <a href="https://github.com/wanderlast/liferay-portal/commit/c3c6f2d59df506ce3a717ba3d1965b696d85a9f0">c3c6f2d59df506ce3a717ba3d1965b696d85a9f0</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-33.liferay.com/job/test-portal-source-format/2829/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/2829/jenkins-report.html">here</a>.</h5></details></html>

wanderlast commented:
ci:test:playwright

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>playwright</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/5067/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5067/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:playwright - 23 out of 27 jobs passed in 2 hours 41 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a3f4976407775eab5d00ef86db452121dab1414d">a3f4976407775eab5d00ef86db452121dab1414d</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a3f4976407775eab5d00ef86db452121dab1414d">a3f4976407775eab5d00ef86db452121dab1414d</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2558548549">EE Development Acceptance (master) - 474 - 2024-04-04[08:45:53]</a></p><details><summary><strong>ci:test:playwright - 23 out of 27 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/5067/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>playwright-compile-jdk8<ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/326882/">playwright-compile-jdk8/0/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/411412/">playwright-js-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/359561/">playwright-js-tomcat90-mysql57-jdk8/1/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/375380/"><strike>playwright-js-tomcat90-mysql57-jdk8/2/0</strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/411411/">playwright-js-tomcat90-mysql57-jdk8/3/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/368461/">playwright-js-tomcat90-mysql57-jdk8/4/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/368462/">playwright-js-tomcat90-mysql57-jdk8/5/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1076962/"><strike>playwright-js-tomcat90-mysql57-jdk8/6/0</strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/341206/">playwright-js-tomcat90-mysql57-jdk8/7/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/798720/">playwright-js-tomcat90-mysql57-jdk8/8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/355784/">playwright-js-tomcat90-mysql57-jdk8/9/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/414646/">playwright-js-tomcat90-mysql57-jdk8/10/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/606954/">playwright-js-tomcat90-mysql57-jdk8/11/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358329/">playwright-js-tomcat90-mysql57-jdk8/12/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/874785/">playwright-js-tomcat90-mysql57-jdk8/13/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/359562/">playwright-js-tomcat90-mysql57-jdk8/14/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/798721/">playwright-js-tomcat90-mysql57-jdk8/15/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/411413/">playwright-js-tomcat90-mysql57-jdk8/16/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/382323/">playwright-js-tomcat90-mysql57-jdk8/17/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/874784/">playwright-js-tomcat90-mysql57-jdk8/18/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358328/"><strike>playwright-js-tomcat90-mysql57-jdk8/19/0</strike></a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/355697/">playwright-js-tomcat90-mysql57-jdk8/20/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/355158/">playwright-js-tomcat90-mysql57-jdk8/21/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/374674/">playwright-js-tomcat90-mysql57-jdk8/22/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/849989/">playwright-js-tomcat90-mysql57-jdk8/23/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/388119/">playwright-js-tomcat90-mysql57-jdk8/24/0</a></li></ul></li></ul></div><details><summary><strong>23 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5067/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/375380//consoleText">playwright-js-tomcat90-mysql57-jdk8/2/0</a> - <a href="https://testray.liferay.com/reports/production/logs/2024-04/test-1-25/test-portal-acceptance-pullrequest(master)/5067/playwright-js-tomcat90-mysql57-jdk8/2/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/375380/testReport/commerce_commerceProducts.spec/ts">commerce/commerceProducts.spec.ts</a></summary><div><ol><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/375380//testReport/commerce_commerceProducts.spec/ts/LPD_5780_modal_title_and_product_name_appear_properly_in_product_menu">LPD-5780 modal title and product name appear properly in product menu</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1076962//consoleText">playwright-js-tomcat90-mysql57-jdk8/6/0</a> - <a href="https://testray.liferay.com/reports/production/logs/2024-04/test-1-25/test-portal-acceptance-pullrequest(master)/5067/playwright-js-tomcat90-mysql57-jdk8/6/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1076962/testReport/frontend-data-set-views-web_creationActions.spec/ts">frontend-data-set-views-web/creationActions.spec.ts</a></summary><div><ol><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1076962//testReport/frontend-data-set-views-web_creationActions.spec/ts/Create_Creation_Actions_on_a_Data_Set_and_show_them_in_a_fragment___Sample_data_clean_up">Create Creation Actions on a Data Set and show them in a fragment › Sample data clean up</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1076962//testReport/frontend-data-set-views-web_creationActions.spec/ts/Create_Creation_Actions_on_a_Data_Set_and_show_them_in_a_fragment___Show_Creation_Actions_menu_in_fragment">Create Creation Actions on a Data Set and show them in a fragment › Show Creation Actions menu in fragment</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1076962//testReport/frontend-data-set-views-web_creationActions.spec/ts/Create_Creation_Actions_on_a_Data_Set_and_show_them_in_a_fragment___Show_a_single_Creation_Action_in_fragment">Create Creation Actions on a Data Set and show them in a fragment › Show a single Creation Action in fragment</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1076962/testReport/frontend-data-set-views-web_fields.spec/ts">frontend-data-set-views-web/fields.spec.ts</a></summary><div><ol><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1076962//testReport/frontend-data-set-views-web_fields.spec/ts/Add_fields_to_a_view_and_show_them_in_a_fragment___Show_mapped_hierarchical_Fields_in_fragment">Add fields to a view and show them in a fragment › Show mapped hierarchical Fields in fragment</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358328//consoleText">playwright-js-tomcat90-mysql57-jdk8/19/0</a> - <a href="https://testray.liferay.com/reports/production/logs/2024-04/test-1-25/test-portal-acceptance-pullrequest(master)/5067/playwright-js-tomcat90-mysql57-jdk8/19/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358328/testReport/knowledge-base-web_knowledge-base-web.spec/ts">knowledge-base-web/knowledge-base-web.spec.ts</a></summary><div><ol><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358328//testReport/knowledge-base-web_knowledge-base-web.spec/ts/can_delete_all_articles_with_a_recycle_bin_disabled">can delete all articles with a recycle bin disabled</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358328//testReport/knowledge-base-web_knowledge-base-web.spec/ts/can_delete_all_articles_with_a_recycle_bin_enabled">can delete all articles with a recycle bin enabled</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358328//testReport/knowledge-base-web_knowledge-base-web.spec/ts/can_publish_and_delete_an_article_with_scheduling_disabled">can publish and delete an article with scheduling disabled</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358328//testReport/knowledge-base-web_knowledge-base-web.spec/ts/can_publish_and_delete_an_article_with_scheduling_enabled">can publish and delete an article with scheduling enabled</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358328//testReport/knowledge-base-web_knowledge-base-web.spec/ts/can_schedule_and_delete_an_article_with_scheduling_enabled">can schedule and delete an article with scheduling enabled</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/5067/">test-portal-acceptance-pullrequest(master)#5067</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5067/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/wanderlast/liferay-portal/pull/109">wanderlast#109</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2558700870">[master] ci:test:playwright - wanderlast &gt; wanderlast - PR#109 - 2024-04-04[15:49:37]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>2558700870<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-25.liferay.com/job/publish-testray-report/10593/">publish-testray-report#10593</a><br/></div></div>
